### ISSUE TITLE
No entries shown on different calendars

### ISSUE THREAD

**Issue by ameierme** [nextcloud/calendar#1926]
You only see entries in the calendar when you go down to daily look. Weekly und monthly entries stay empty.

**Comment by georgehrke** [comment]
please use the [issue template](https://raw.githubusercontent.com/nextcloud/calendar/master/.github/ISSUE_TEMPLATE/bug.md) as described in our [contributing guidelines](https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md)

Unless you fill out the issue template completely including all logs, we can't go ahead and help you.

**Comment by ameierme** [comment]
Log says: Use of undefined constant calendar - assumed 'calendar' (this will throw an Error in a future version of PHP) at /mnt/web110/d0/59/53201359/htdocs/nextcloud/config/config.php#23

**Comment by tcitworld** [comment]
Seems like an issue with your install. What's in `config/config.php` line 23 (anonymize if needed)?

**Comment by ameierme** [comment]
'defaultapp' => calendar,
But i think the error is somewhere else:
[PHP] Error: Use of undefined constant calendar - assumed 'calendar' (this will throw an Error in a future version of PHP) at /mnt/web110/d0/59/53201359/htdocs/nextcloud/config/config.php#23

Error | PHP | opendir(/var/tmp/): failed to open dir: Permission denied at /mnt/web110/d0/59/53201359/htdocs/nextcloud/lib/private/TempManager.php#190
-- | -- | --

**Comment by raimund-schluessler** [comment]
> 'defaultapp' => calendar,

`calendar` needs to be in quotes, otherwise it's not a string but an "undefined constant calendar", just as the error message says.

**Comment by ameierme** [comment]
Quotes are set, Erroressage is gone. But still no entries visible

**Comment by georgehrke** [comment]
Please fill the __entire__ issue template like I already requested [above](https://github.com/nextcloud/calendar/issues/1926#issuecomment-579227400).

**Comment by ameierme** [comment]
---
name: 🐛 Bug
about: Have you encountered a bug?
title: ''
labels: bug, 0. to triage
assignees: ''
version: 0.1
---

 

### Steps to reproduce
1. Log into Nextcloud to show calendar
2. Outlook and Thunderbird shows entries
3. No Entries in Browser

### Expected behaviour
Should see calendar entries

### Actual behaviour
Calender stays empty. 

### Calendar app
2.01

**CalDAV-clients used:** Thunderbird and Outlook
### Client configuration
**Browser:** Chrome)

**Operating system:** Windows 10

### Server configuration

**Operating system**: Hosted on Strato

**Web server:** Hosted on STRATO

**Database:** Maria-DB

**PHP version:** 7.3

**Nextcloud Version:** 18.0

**Updated from an older installed version or fresh install:** Update

**List of activated apps:**
	1.4.0	Offiziell	
Activity	2.11.0	Offiziell	
Auditing / Logging	1.8.0	Offiziell	
Brute-force settings	1.5.0	Offiziell	
Calendar	2.0.1	Offiziell	
Collaborative tags	1.8.0	Offiziell	
Comments	1.8.0	Offiziell	
Contacts	3.1.8	Offiziell	
Deleted files	1.8.0	Offiziell	
External storage support	1.9.0	Offiziell	
Federation	1.8.0	Offiziell	
File sharing	1.10.1	Offiziell	
First run wizard	2.7.0	Offiziell	
Log Reader	2.3.0	Offiziell	
Monitoring	1.8.0	Offiziell	
Nextcloud announcements	1.7.0	Offiziell	
Notifications	2.6.0	Offiziell	
OCC Web	0.0.6		
ONLYOFFICE	4.1.4	Offiziell	
Password policy	1.8.0	Offiziell	
PDF viewer	1.7.0	Offiziell	
Photos	1.0.0	Offiziell	
Privacy	1.2.0	Offiziell	
Recommendations	0.6.0	Offiziell	
Right click	0.15.2	Offiziell	
Share by mail	1.8.0	Offiziell	
Support	1.1.0	Offiziell	
Text	2.0.0	Offiziell	
Theming	1.9.0	Offiziell	
Update notification	1.8.0	Offiziell	
Usage survey	1.6.0	Offiziell	
Versions	1.11.0	Offiziell	
Video player	1.7.0	Offiziell	
Default encryption module	2.6.0	Offiziell	
LDAP user and group backend

```
If you have access to your command line run e.g.:
- accessibility: 1.4.0  - activity: 2.11.0  - admin_audit: 1.8.0  - bruteforcesettings: 1.5.0  - calendar: 2.0.1  - cloud_federation_api: 1.1.0  - comments: 1.8.0  - contacts: 3.1.8  - dav: 1.14.0  - federatedfilesharing: 1.8.0  - federation: 1.8.0  - files: 1.13.1  - files_external: 1.9.0  - files_pdfviewer: 1.7.0  - files_rightclick: 0.15.2  - files_sharing: 1.10.1  - files_trashbin: 1.8.0  - files_versions: 1.11.0  - files_videoplayer: 1.7.0  - firstrunwizard: 2.7.0  - logreader: 2.3.0  - lookup_server_connector: 1.6.0  - nextcloud_announcements: 1.7.0  - notifications: 2.6.0  - oauth2: 1.6.0  - occweb: 0.0.6  - onlyoffice: 4.1.4  - password_policy: 1.8.0  - photos: 1.0.0  - privacy: 1.2.0  - provisioning_api: 1.8.0  - recommendations: 0.6.0  - serverinfo: 1.8.0  - settings: 1.0.0  - sharebymail: 1.8.0  - support: 1.1.0  - survey_client: 1.6.0  - systemtags: 1.8.0  - text: 2.0.0  - theming: 1.9.0  - twofactor_backupcodes: 1.7.0  - updatenotification: 1.8.0  - viewer: 1.2.0  - workflowengine: 2.0.0Disabled:  - encryption  - user_ldap
from within your instance's installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
{    "system": {        "instanceid": "***REMOVED SENSITIVE VALUE***",        "passwordsalt": "***REMOVED SENSITIVE VALUE***",        "secret": "***REMOVED SENSITIVE VALUE***",        "trusted_domains": [            "kalender.stmarien-wassenberg.de"        ,        "datadirectory": "***REMOVED SENSITIVE VALUE***",        "dbtype": "mysql",        "version": "18.0.0.10",        "overwrite.cli.url": "https:\/\/kalender.stmarien-wassenberg.de",        "dbname": "***REMOVED SENSITIVE VALUE***",        "dbhost": "***REMOVED SENSITIVE VALUE***",        "dbport": "",        "dbtableprefix": "oc_",        "mysql.utf8mb4": true,        "dbuser": "***REMOVED SENSITIVE VALUE***",        "dbpassword": "***REMOVED SENSITIVE VALUE***",        "installed": true,        "force_language": "de",        "defaultapp": "calendar",        "loglevel": 1    }}]

### Logs
#### Web server error log (e.g. /var/log/apache)
```
Insert your webserver log here
```

#### Log file (data/nextcloud.log)
```

Error | PHP | opendir(/var/tmp/): failed to open dir: Permission denied at /mnt/web110/d0/59/53201359/htdocs/nextcloud/lib/private/TempManager.php#190
-- | -- | --




#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

**Comment by georgehrke** [comment]
Please provide the requested browser logs as well ...

**Comment by ameierme** [comment]
[Firefox Browser Log.docx](https://github.com/nextcloud/calendar/files/4127663/Firefox.Browser.Log.docx)

**Comment by georgehrke** [comment]
@ameierme If the log is too long, can you please paste them in a gist? https://gist.github.com

**Comment by ameierme** [comment]
File is uploaded https://github.com/nextcloud/calendar/files/4127663/Firefox.Browser.Log.docx

**Comment by traits** [comment]
Had the same issue, after updating from 17.0.1 to 17.0.3. Only drilling into day view (what I usually only in very rare cases do) entries appeared in weekly and even monthly view again. Before doing this even logging out didn't change anything.

**Comment by ameierme** [comment]
Seems to me it's the calender-App not the Nextcloud-Version. As from calender 2.0.1 the problem occurs

**Comment by tcitworld** [comment]
@ameierme This is not the javascript browser log. Open the developer console with the F12 tab, then click on Console tab, access the calendar and copy what's in the console here or in a pastebin or gist like @georgehrke said.

@traits Please provide javascript browser logs as well.

**Comment by traits** [comment]
Here we go:

FF 72.0.2 (Win 10)

`Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). injected.js:1:5301
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. injectStylesIntoStyleTag.js:201:10
oc_config is deprecated: use OC.config instead globals.js:26:15
Proxying an event bus of version 0.2.1 with 1.1.2 ProxyBus.js:74:14
Initial settings: 
Object { appVersion: "2.0.1", firstRun: false, showWeekends: true, showWeekNumbers: true, skipPopover: false, talkEnabled: false, timezone: "automatic" }
settings.js:87:2
[DEBUG] calendar: The calendar app is using version 2.2019c of the timezone database 
Object { app: "calendar", uid: "authenticated:micha" }
ConsoleLogger.js:44:18
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
Fallback to locale en moment.js:83:3
[DEBUG] calendar: Current user principal is /remote.php/dav/principals/users/micha/ 
Object { app: "calendar", uid: "authenticated:micha" }
ConsoleLogger.js:44:18
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
too much recursion InternalError: "too much recursion"
    t lockable.js:35
    t observer.js:34
    t abstractValue.js:41
    t calendar.js:2
    value dateTimeValue.js:414
    value dateTimeValue.js:404
    get eventComponent.js:86
    value eventComponent.js:51
    get eventComponent.js:90
    value eventComponent.js:51
    get eventComponent.js:90
    value eventComponent.js:51
main.esm.js:5221:16

`

**Comment by georgehrke** [comment]
@ameierme Please provide logs in a valid format, which is either paste them in here or preferably a gist.

Microsoft word documents __are not__ accepted formats for logs.

**Comment by georgehrke** [comment]
@traits We fixed a similar error message in 2.0.1 of the calendar app. Please make sure you are running the latest version. If that still doesn't work, please open a separate issue filling the entire issue template. thx.
